db/migrate/20210304015405_add_timestamps_to_access_tokens.rb

```diff
@@ -0,0 +1,10 @@
+class AddTimestampsToAccessTokens < ActiveRecord::Migration[6.0]
+  def change
+    add_timestamps :email_access_tokens, default: Time.now
+    add_timestamps :text_message_access_tokens, default: Time.now
+    change_column_default :text_message_access_tokens, :created_at, to: nil, from: Time.now
```

Comment:
will this invalidate all current access tokens? Or not affect "existing" access tokens at all. I'm not sure how SQL treats nil values when doing comparsions like "created_at > ?", Time.current - 2 days...

Maybe we're not using these tables yet so thats why it doesn't affect them? 

Also, why do we want these default values to be nil?

Reply:
Here's the sequence of operations:

* `add_timestamps` creates columns (which are `NOT NULL` in the DB) - this _would_ fail for existing rows, except that I also set a database-level default, so existing rows get timestamps (of the time this migration runs)
* `change_column_default` removes the DB-level default

This means all existing tokens are valid for 2 days from when the migration runs.

The default value isn't really nil; it's really no default at all; that's the common pattern for these timestamp columns (check out schema.rb for other examples).

So... this was a bit of juggling to jam into one migration, but it seems to work.